### PR TITLE
add channel handling scripts

### PR DESCRIPTION
**Pull request by szuecs**
add channel handling scripts

### CODE REVIEW

**Comment by njuettner** [comment]
👍

**Comment by aermakov-zalando** [comment]
Can you add a hashbang and `set -e`? This will happily continue on errors.

File: alpha-to-beta.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+git co alpha
```
Comment (mikkeloscar) [inline alpha-to-beta.sh]: I think `co` is an alias? Please spell out the full command `checkout` :)

Same for `br` -> `branch` etc.

**Comment by szuecs** [comment]
I added a bit more automation
* prepares PR message and title
* if "hub" is installed then it creates a PR with title and body and adds our testing label automatically

**Comment by szuecs** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
Thanks!

Shellcheck prints some warnings, at least one of those looks that it matters:

    In alpha-to-beta.sh line 16:
    echo "alpha-to-beta\n"> $MSG
          ^-- SC2028: echo won't expand escape sequences. Consider printf.

The rest looks relatively benign, but I'd still fix the reported issues.

Additionally, the scripts only differ in the branch names. Would you consider having a `prepare-pr` script instead that would accept source and target branches as arguments?

**Comment by szuecs** [comment]
@aermakov-zalando changed as commented

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**